IWXXM/ReleaseNotes-IWXXM.txt

```diff
@@ -1,5 +1,35 @@
 =========================================
-2023-1 - 15 June 2023
+2025-2 Release Candidate 1 - Feb 2025
+=========================================
+Package METAR/SPECI (Version 3.2.0RC1)
+  * Removed iwxxm:runwayState, iwxxm:AerodromeRunwayState, iwxxm:RunwayDeposits, iwxxm:RunwayContamination, and
+    iwxxm:RunwayFrictionCoefficient, as well as schematron rules METAR_SPECI.AerodromeRunwayState-1 to 3
+  * Revised the multiplicity of iwxxm:rvr to [0..*]
+  * Updated the notes on the possibility to provide temperature and dew-point temperature values in tenths of a degree
+    Celsius
+Package Common (Version 3.2.0RC1)
+  * Removed iwxxm:Runway
```

Comment:
```suggestion
  * Removed aixm:Runway
```

Reply:
Yes and no.  We are "stealing" aixm:Runway and make it available under our own namespace, viz iwxxm:Runway.  You can see this in our examples (like https://schemas.wmo.int/iwxxm/2023-1/examples/metar-A3-1.xml).

Reply:
I don't understand. "<complexType name="RunwayPropertyType"><sequence minOccurs="0"><element ref="aixm:Runway"/>..."
was removed from common.xsd. So to me, it doesn't look like "iwxxm:Runway" was there before and therefore couldn't be removed....?

Reply:
This is a long story and I don't think the explanation exists on GitHub issues.  In the original AIXM schema it does not accept the use of attribute nilReason which can be an xlink making reference to https://codes.wmo.int/common/nil.  Instead, it only accepts certain strings as indicated in [its documentation](https://ext.eurocontrol.int/aixm_confluence/display/ACG/Use+of+nilReason).  To do this, we redefine the respective AIXM elements by using their original types but different gml:AssociationAttributeGroup and gml:OwnershipAttributeGroup.  This trick has been mentioned to AIXM folks but since they have no intention to update AIXM to meet our needs they have no further comment to what we have done.